templates/burger-menu.twig

```diff
@@ -60,23 +60,22 @@
 					{% include "burger-menu-items.twig" with { menu: navbar_menu_items } %}
 				{% endif %}
 
-				{% if donate_menu_items is not empty and donate_menu_items[0].children|length > 1 and dropdown_menu %}
+				{% if donate_menu_items is not empty and donate_menu_items[0].children|length >= 1 and dropdown_menu %}
 					{% include "burger-menu-items.twig" with { menu: donate_menu_items } %}
 				{% endif %}
 			</ul>
 		</nav>
 	</div>
 
-	{% if donate_menu_items|first is not null %}
-		{% set donate_menu = donate_menu_items|first %}
+	{% if donate_menu_default|first is not null %}
 		<div class="burger-menu-footer">
 			<a
-					class="btn btn-donate"
-					href="{{ donate_menu.link }}"
-					data-ga-category="Menu Navigation"
-					data-ga-action="Donate"
-					data-ga-label="{{ page_category }}">
-				{{ __( 'Make a donation', 'planet4-master-theme' ) }}
+				class="btn btn-donate"
```

Comment:
The button "Donate" style will be removed, and these now use our primary style... We can keep `btn-donate`, especially for child themes, but maybe we should use `btn-primary` too? 🤔  Right now [this is duplicating some primary button styles](https://github.com/greenpeace/planet4-master-theme/blob/6923bd518aa3597b1686803e9719d87958a59f22/assets/src/scss/layout/_navbar.scss#L57-L69)
Could be done in a separate PR though 🙂

Reply:
If we change to `btn-primary`, we will have also to change something else from [here](https://github.com/greenpeace/planet4-master-theme/blob/278c0f242bb25bb646e78b9225d2533b0bb7f86b/assets/src/scss/components/_buttons.scss#L174-L176) and that styles are also being with other components. So, I agree to do it in a separate PR whenever we'll work on this removal.